### PR TITLE
feat: [FC-0006] add Verifiable Credentials feature

### PR DESCRIPTION
**Pull request by wowkalucky**
**Note:**  for the sake of review effort simplification, it is decided to split this large PR onto 2 parts: 
- initial part - introduces a feature flag with a new application modules structure
- main part - adds all the implementation logic

This is the part 2/2 of the Verifiable Credentials feature.

Related PRs:

- [1nd part](https://github.com/openedx/credentials/pull/1973)
- [documentation](https://github.com/openedx/credentials/pull/1991)
- [configuration](https://github.com/openedx/configuration/pull/6928)

----

**Run JavaScript tests locally with Karma**

There is work being done on a fix to get Karma to run in CI. Until then, however, contributors are required to run these tests locally.

- [x] Make sure you are inside the devstack container
- [x] Run `make test-karma`
- [x] All tests pass

---

- Relates to https://github.com/openedx/frontend-app-learner-record/pull/151

## TODO

- [x] - add related Open edX Credentials PR link.
- [x] - update test coverage

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @wowkalucky!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: credentials/apps/verifiable_credentials/issuance/models.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        )
+        if hash_str is None:
+            return status_list_url
+
+        return f"{status_list_url}#{hash_str}"
+
+    @classmethod
+    def resolve_issuer(cls):
+        """
+        Unconditionally (for now) returns system-level Issier ID.
```
Comment (GlugovGrGlib) [inline credentials/apps/verifiable_credentials/issuance/models.py:132]: Typo in a word Issuer

**Comment by wowkalucky** [comment]
@GlugovGrGlib updated for review (coverage check is broken for some reason).

**Comment by wowkalucky** [comment]
@GlugovGrGlib rebased to master

**Review by GlugovGrGlib — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@wowkalucky 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/credentials&entry.867055334=https://github.com/openedx/credentials/pull/2001&entry.1484655318=https://github.com/wowkalucky&entry.752974735=2023-05-08+14:39&entry.1917517419=2023-07-21+07:43&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**